core/src/com/unciv/ui/cityscreen/CityStatsTable.kt

```diff
@@ -34,10 +36,11 @@ class CityStatsTable(val cityScreen: CityScreen): Table() {
         outerPane = ScrollPane(innerTable)
         outerPane.setOverscroll(false, false)
         outerPane.setScrollingDisabled(true, false)
-        add(outerPane).maxHeight(cityScreen.stage.height / 2)
+        statsCell = add(outerPane)
+        statsCell.maxHeight(cityScreen.stage.height / 2)
```

Comment:
Likely redundant

Reply:
You mean setting maxHeight during init()? Are we guaranteed update()?

Reply:
Exactly - are we?

Reply:
Functionally my guess is yes to populate the tables with values. But unsure. Likely safe to remove

Reply:
If after analysis or debugging you're still unsure, leave it. If you're sure, ditch. Me, I just like to prod a little.